astropy/io/misc/asdf/tags/transform/functional_models.py

```diff
@@ -0,0 +1,82 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from numpy.testing import assert_array_equal
+
+from asdf import yamlutil
+
+from astropy import modeling
+from .basic import TransformType
+from . import _parameter_to_value
+
+
+__all__ = ['Gaussian1DType', 'Gaussian2DType']
+
+
+class Gaussian1DType(TransformType):
+    name = "transform/gaussian1d"
+    version = '1.0.0'
+    types = ['astropy.modeling.functional_models.Gaussian1D']
+
+    @classmethod
+    def from_tree_transform(cls, node, ctx):
+        return modeling.functional_models.Gaussian1D(node['amplitude'],
```

Comment:
I would use keyword parameters here, since the order of amplitude, mean, and stddev is defined by the order of the parameter definitions in the class, and not an explicit `__init__` method -- it seems like it'd be easy for someone to switch them around without realizing that they changed the interface.